web-rdfs/source/http_handlers.cc

```diff
@@ -19,18 +26,61 @@ void append_file_handler(std::shared_ptr<HttpsServer::Response> response,
   response->write("append_file_handler");
 }
 
+void delete_file_handler(std::shared_ptr<HttpsServer::Response> response,
+                         std::string path) {
+  LOG(DEBUG) << "HTTP request: delete_file_handler";
+
+  // TODO(security): implement
+  hadoop::hdfs::DeleteResponseProto res;
+  hadoop::hdfs::DeleteRequestProto req;
+
+  req.set_src(path);
+  zkclient::ZkNnClient::DeleteResponse zkResp = zk->destroy(req, res);
+
+  response->write(webRequestTranslator::getDeleteResponse(zkResp));
+}
+
 void read_file_handler(std::shared_ptr<HttpsServer::Response> response,
-                         std::shared_ptr<HttpsServer::Request> request) {
+                         std::string path) {
   LOG(DEBUG) << "HTTP request: read_file_handler";
 
-  // TODO(security): implement
-  response->write("read_file_handler");
+  std::string storedFile = "tempStore" + path;
+  std::string input = "hdfs dfs -fs hdfs://localhost:5351 -cat /" + path +
```

Comment:
This feels very fragile. I'm guessing there is no way to invoke this directly? What happens if the RDFS port changes to something other than 5351?

Reply:
I honestly don't know if there is anyway to make this less sketchy